boards/arm/96b_carbon/doc/96b_carbon.rst

```diff
@@ -281,6 +310,50 @@ terminal:
 
    Hello World! arm
 
+.. _96b_carbon_verify_bluetooth:
+
+Verifying Bluetooth Functionality
+---------------------------------
+
+This section contains instructions for verifying basic Bluetooth
+functionality on the board. For help on Zephyr applications
+in general, see :ref:`build_an_application`.
+
+1. Flash the nRF51 with the hci_spi sample application as described in
+   :ref:`96b_carbon_nrf51_bluetooth`.
+
+2. Install the dfu-util flashing app, as described above.
+
+3. Build the ``samples/bluetooth/ipsp`` application for 96b_carbon::
```

Comment:
Well, I agree that reusable doc parts should be factored out and linked to from the relevant places. Just as in general, docs need more work. But in the meantime, Networking/BLE support for 96b_carbon is in works for ~1year, and a lot of people wait for it exactly as it allows networking connectivity. So perhaps, docs refactoring can be left for another patch (this one is already big at 8 commits).